<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addhw</class>
 <widget class="QWidget" name="addhw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>homework3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>homework2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>add homwwork</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="addhw3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="addhw2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="addhw1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>homework1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
